test/html.js

```diff
@@ -319,14 +319,26 @@ describe('HTML Parser', function () {
 			});
 		});
 
+		describe('#getAttribute', function () {
```

Comment:
With this method, Maybe we should provide getter `rawAttributes` to end uses.

Reply:
Yes, I also thought about it, but it's not it's not to hard to do `rawAttributes [attr]` anyway who really needs raw values. I added `getAttribute` to "standardize". So now the same code with `getAttribute` works in browsers too.